packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx

```diff
@@ -40,6 +48,10 @@ type VisualizationContext = {
     isLoading: boolean;
     columnOrder: string[];
     isSqlRunner: boolean;
+    dimensions: Dimension[];
+    metrics: Metric[];
+    customMetrics: Metric[];
```

Comment:
technically it can remain a Metric since I just want a subset and `AdditionalMetric` extends `Metric` but I'll make a change for readability and consistency

Reply:
sorry I was wrong here. Metric extends Field. AdditionalMetric does not extend Metric even tho it has quite a good amount of the same properties. updated type. just wanted to clarify.